pkgs/development/python-modules/pyu2f/default.nix

```diff
@@ -36,8 +36,8 @@ buildPythonPackage rec {
     do
       # https://docs.python.org/3/whatsnew/3.12.html#id3
       substituteInPlace pyu2f/tests/$path \
-        --replace "assertEquals" "assertEqual" \
-        --replace "assertRaisesRegexp" "assertRaisesRegex"
+        --replace-quiet "assertEquals" "assertEqual" \
```

Comment:
Since we have a list of files here, we might as well clean out the list and use replace-fail

Reply:
True

Reply:
Fixed